mindmeld/components/dialogue.py

```diff
@@ -906,9 +911,7 @@ def _retry_logic(self, responder, nlr):
             self._app.app_manager.dialogue_manager.reprocess()
 
     def __call__(self, request, responder):
-        """The iterative call to fill missing slots in the entity form till all slots have been
-        filled up or the flow has been exited.
-
+        """
```

Comment:
why remove doc?

Reply:
Just removed the first half. It's been moved to the individual call functions.